examples/chip-tool-darwin/commands/clusters/WriteAttributeCommandBridge.h

```diff
@@ -47,6 +47,7 @@ class WriteAttribute : public ModelCommand {
     WriteAttribute(const char * _Nonnull attributeName)
         : ModelCommand("write")
     {
+        AddArgument("data-version", 0, UINT32_MAX, &mDataVersion);
```

Comment:
Similarly, write-by-id should allow this?

Reply:
I think it should also be included in the write-by-id, but the CHIP.framework does not support passing in this.

Reply:
Again, please file issues as needed.